packages/@webex/plugin-meetings/test/unit/spec/meeting/muteState.js

```diff
@@ -509,3 +509,221 @@ describe('plugin-meetings', () => {
+        const serverMutedSpyCall2 = setServerMutedSpy.getCall(1);
+
+        assert.calledOnceWithExactly(applyUnmuteAllowedToTrackSpy, meeting);
+        assert.calledOnceWithExactly(muteLocalTrackSpy, meeting, true, 'remotelyMuted');
+        assert.equal(muteState.ignoreMuteStateChange, false);
+        assert.equal(serverMutedSpyCall1.calledWithExactly(true, 'remotelyMuted'), true);
+        assert.equal(serverMutedSpyCall2.calledWithExactly(true, 'remotelyMuted'), true);
+        assert.calledOnceWithExactly(applyClientStateToServerSpy, meeting);
+        assert.isFalse(muteState.state.client.localMute);
+      });
+
+      it('#muteLocalTrack tracks undefined', async () => {
+        setup(mediaType, true, false, false);
+        setupSpies(muteState, mediaType);  
+        muteState.init(meeting);
+        assert.calledOnceWithExactly(applyUnmuteAllowedToTrackSpy, meeting);
+        assert.calledOnceWithExactly(muteLocalTrackSpy, meeting, true, 'remotelyMuted');
+        assert.equal(muteState.ignoreMuteStateChange, false);
+      });
+
+      describe('#handleLocalTrackMuteStateChange', () => {
+
+        afterEach(() => {
+          sinon.restore();
+        });
+
+        it('checks when ignoreMuteStateChange is true', () => {
+          setup(mediaType);
+          muteState.ignoreMuteStateChange= true;
+          muteState.state.client.localMute = false;
+
+          const spy = sinon.spy(muteState, 'applyClientStateToServer');
+          muteState.handleLocalTrackMuteStateChange(meeting, true);
+          assert.notCalled(spy);
+          assert.isFalse(muteState.state.client.localMute);
+        });
+
+        it('tests localMute - true to false', () => {
+          setup(mediaType);
+          muteState.state.client.localMute = true;
+
+          const spy = sinon.spy(muteState, 'applyClientStateToServer');
+          muteState.handleLocalTrackMuteStateChange(meeting, false);
+          assert.equal(muteState.state.client.localMute, false);
+          assert.calledOnceWithExactly(spy, meeting)
+        });
+
+        it('tests localMute - false to true', () => {
+          muteState.state.client.localMute = false;
+
+          const spy = sinon.spy(muteState, 'applyClientStateToServer');
+          muteState.handleLocalTrackMuteStateChange(meeting, true);
+          assert.equal(muteState.state.client.localMute, true);
+          assert.calledOnceWithExactly(spy, meeting)
+        });
+      });
+
+      describe('#applyClientStateLocally', () => {
+
+        afterEach(() => {
+          sinon.restore();
+        });
+
+        it('checks when ignoreMuteStateChange is false', () => {
+          setup(mediaType);
+          setupSpies(muteState, mediaType);
+          muteState.sdkOwnsLocalTrack= false;
+
+          muteState.applyClientStateLocally(meeting, 'somereason');
+          assert.calledOnceWithExactly(muteLocalTrackSpy, meeting, muteState.state.client.localMute, 'somereason');
```

Comment:
we should be checking that setMuted was called on the track instead of a private muteLocalTrack